sphinx/source/developers/ledger.rst

```diff
@@ -5,20 +5,18 @@ The ledger is the persistent distributed append-only record of the transactions
 
 A node writes its ledger to a file as specified by the ``--ledger-file`` command line argument.
 
-Ledger encryption
+Ledger Encryption
 -----------------
 
 Each entry in the ledger corresponds to a transaction (or delta) committed by the primary's key-value store.
 
 When a transaction is committed, each affected ``Store::Map`` is serialised in different security domains (i.e. public or private), based on the policy set when the ``Store::Map`` was created (default is private). Public ``Store::Map`` are serialised and stored in the ledger as plaintext while private ``Store::Map`` are serialised and encrypted before being stored.
 
-Note that even if a transaction only affects a private ``Store::Map``, unencrypted information such as the version number is always present in the serialised entry. More information about the ledger entry format is available in the :ref:`protocol` section.
+Ledger entries are integrity-protected and encrypted using a symmetric key (AES256-GCM) shared by all trusted nodes. This key is kept secure inside each enclave and sealed to disk for service recovery. See :ref:`members/common_member_operations:Rekeying Ledger` for details on how members can rotate the ledger encryption key.
```

Comment:
This has been added wrt rekeying.

Reply:
We have a page about crypto/algos, I think this should link there and should avoid mentioning AES-GCM (not that we have any plans to change, but just to keep this centralised/agile).